docs/migration-guide.md

```diff
@@ -27,11 +27,20 @@ The type `sds` is removed from the public API.
 
 ## Migrating from `hiredis-cluster` 0.14.0
 
+* The cluster client initiation procedure is changed and `valkeyClusterOptions`
+  should be used to specify options when creating a context.
+  See documentation for configuration examples when using the
+  [Synchronous API](cluster.md#connection-options) or the
+  [Asynchronous API](cluster.md#connecting-1).
```

Comment:
Why not link to the headings mentioned in the link text?

How to connect is the first subsection within these sections anyway.

```suggestion
  [Synchronous API](cluster.md#synchronous-api) or the
  [Asynchronous API](cluster.md#asynchronous-api).
```

Reply:
Sure